zwave_js_server/model/controller/__init__.py

```diff
@@ -646,9 +648,12 @@ def receive_event(self, event: Event) -> None:
                 f"Controller doesn't know how to handle/forward this event: {event.data}"
             )
 
-        validate_event_data(
-            event.data, "controller", event.type, CONTROLLER_EVENT_MODEL_MAP
-        )
+        if event.type not in CONTROLLER_EVENT_MODEL_MAP:
+            raise TypeError(f"Unknown controller event type: {event.type}")
+        try:
+            CONTROLLER_EVENT_MODEL_MAP[event.type](**event.data)
+        except ValidationError as exc:
```

Comment:
Do we even need to catch these errors? Maybe it's enough to just let them be raised as is?

Reply:
I guess not!